### PR TITLE
Fixed js formatting

### PR DESCRIPTION
**Pull request by prashantiA**
Formatting fixes on js code using eslint

### CODE REVIEW

**Comment by prashantiA** [comment]
@nitinramadoss those are the onload functions which are called in index.html. In addition, its complaining that google is undefined when it shouldn't be. Looking into how to configure the linter to not complain about these specific cases as well as figuring out what's causing the exit status 1 on the java formatter before I merge this.

**Comment by prashantiA** [comment]
@nitinramadoss can you take another look at this so I can merge it in?

File: run-all-linter-tests.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (lee12jin) [inline run-all-linter-tests.sh:1]: Maybe shorten this name so that it's easy to run

./lint.sh

File: capstone/src/main/webapp/script.js [hunk 2]
```diff
@@ -196,3 +197,6 @@ function loadReponseChart() {
       document.getElementById('response-bar-chart'));
   chart.draw(stats, null);
 }
+
+google.charts.load('current', {packages: ['corechart']});
```
Comment (nitinramadoss) [inline capstone/src/main/webapp/script.js:201]: Placing this code here prevents the charts from loading

Comment (nitinramadoss) [reply capstone/src/main/webapp/script.js:201]: Charts API Docs recommend placing the code in script tags within the HTML file. Is there any way to configure the linter to ignore the HTML files?

Comment (prashantiA) [reply capstone/src/main/webapp/script.js:201]: The HTML files are not the problem. The problem is that I can't mark the loadCharts function as exported so it thinks its unused.

**Pull request merged**